### PR TITLE
feat: added route to upload files to s3 bucket

### PR DESCRIPTION
**Pull request by TarunTomar122**
it works perfectly with our cli <3

### CODE REVIEW

File: dashboard/.env.example [hunk 1]
```diff
@@ -13,4 +13,8 @@ POSTGRES_HOST=database
 
 # PG Admin config
 PGADMIN_DEFAULT_EMAIL=admin@example.com
-PGADMIN_DEFAULT_PASSWORD=adminpassword
\ No newline at end of file
+PGADMIN_DEFAULT_PASSWORD=adminpassword
+
+# AWS config
+AWS_ACCESS_KEY_ID = somekey
```
Comment (rohansingh-chat360) [inline dashboard/.env.example:19]: Upar space nhi hai `=` ke beech me to idhar kyu hai?

File: dashboard/controllers/filesController.go [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+	"github.com/aws/aws-sdk-go/aws/session"
+
+	"github.com/gin-gonic/gin"
+
+	"github.com/aws/aws-sdk-go/service/s3/s3manager"
+
+	"github.com/aws/aws-sdk-go/aws/credentials"
+)
+
+const (
```
Comment (rohansingh-chat360) [inline dashboard/controllers/filesController.go:18]: IDK but the indentation seems to be too big in this file.

File: dashboard/go.mod [hunk 3]
```diff
@@ -1,6 +1,6 @@
 module github.com/rohansingh9001/Neura-Launch-Dashboard
 
-go 1.20
+go 1.21
```
Comment (rohansingh-chat360) [inline dashboard/go.mod:3]: No. Not acceptable. Use GoLang 1.20

**Review by rohansingh-chat360 — CHANGES_REQUESTED** [review]
gofmt exists.

File: dashboard/.env.example (hunk 1 above)
Comment (rohansingh9001) [reply dashboard/.env.example:19]: True

File: dashboard/controllers/filesController.go (hunk 2 above)
Comment (rohansingh9001) [reply dashboard/controllers/filesController.go:18]: Spittin facts

File: dashboard/go.mod (hunk 3 above)
Comment (rohansingh9001) [reply dashboard/go.mod:3]: Shi baat hai

**Comment by TarunTomar122** [comment]
> Fas Fas kardo

Done @rohansingh9001

**Review by rohansingh9001 — APPROVED** [review]
FasFas

Also It seems like Indentation is set at 8 spaces. Pure codebase me we need to make it 4.

**Pull request merged**